### DISCUSSION TITLE
Does this tool work with ArcGIS Pro Basic (not Standard)

### DISCUSSION THREAD

**Discussion by nursha01** [norwegian-geotechnical-institute/REMEDY_GIS_RiskTool#2] [Q&A]
HI 
I would like to use this tool to assess settlement risk for buildings. But when i run Begrens Skade Excavation, it failed to open. What can be the reasons for that? 
Thanks !

**Comment by asgeirlysdahl** [comment]
Hello nursha01, we know that there is a norwegian letter "ø" in one of the tool parameter names. Go to the tool properties, parameters and change this to "o", try that
